website/blog/2018-07-27-removing-babels-stage-presets.md

```diff
@@ -0,0 +1,189 @@
+
+And this project is no stranger to community uproar 🙂, as Babel 6 was a "controversial" release for many. There were plently of posts right after and even [recently](https://news.ycombinator.com/item?id=11371906) that speak about how difficult it was to use.
+
+But removing previous defaults or removing the Stage presets doesn't mean we don't care about ease of use, new users, or documentation. We try with what we can to keep the project stable, document what we know, and provide tooling to make things better.
+
+## Migrating
+
+> For a more automatic migration, we have updated [babel-upgrade](https://github.com/babel/babel-upgrade) to do this for you (you can run `npx babel-upgrade`).
+
+The TL;DR is that we removing the Stage presets. At some level, people will have to opt-in and know what kinds of proposals are being used instead of assuming what people should use, especially given the unstable nature of some of these proposals. But if you use your another preset or a toolchain (i.e. [create-react-app](https://github.com/facebook/create-react-app)) it's possible this change doesn't affect you directly.
+
+We have deprecated the Stage presets as of `7.0.0-beta.52`, so if you don't want to change your config now we would suggest you *pin* the versions to `beta.54` until you can upgrade; after `beta.54` we will throw an error with a message saying how to migrate.
+
+As as alternative, you are free to make your own preset that contains the same plugins and upgrade them as you please (someone make a tool that automatically creates a preset from your config). In the future, we may want to work on a `babel-init` that can help you setup plugins interactively or update `babel-upgrade` itself to accomplish it. Maybe Babel should stay as a low-level tool and rely on other higher-level/framework tools like `create-react-app` to handle these choices for people.
+
+## The Future
+
+### Proposal Implementations in Babel
+
+[Josh Justice](https://twitter.com/CodingItWrong) wrote a [post](https://babeljs.io/blog/2018/07/19/whats-happening-with-the-pipeline-proposal) recently about the changes to using the pipeline operator (`|>`).
+
+The main point there is that the proposal itself is in flux and has a few options to explore. Because we'd like to implement all three of the current possibilities as Babel plugins for both spec feedback and user feedback, we believed the way the plugin is used should change as well.
+
+Before we would simply add the proposal plugin to the config and that was it. Now we would remove the default behavior and ask users to opt-in to a flag that shows which proposal is chosen, and makes it clear that there isn't a fixed or even favored option at the moment.
+
+```diff
+{
+  "plugins": [
+-   "@babel/plugin-proposal-pipeline-operator"
++   ["@babel/plugin-proposal-pipeline-operator", { "proposal": "minimal" }]
+  ]
+}
+```
+
```

Comment:
It might be worth expanding here that this is a new approach for TC39, and this experimentation could inform future proposals.